app/views/links/index.html.erb

```diff
@@ -1,52 +1,45 @@
-<%= render 'projects/header', project: @project %>
-<div class="page-body">
-  <div class="container-xl">
-    <div class="row row-cards">
-      <div class="col-12">
-        <div class="card">
-          <div class="card-header">
-            <h3 class="card-title">Links</h3>
-          </div>
-          <% if @links.any? %>
-            <div class="table-responsive">
-              <table class="table card-table table-vcenter text-nowrap datatable">
-                <thead>
-                  <tr>
-                    <th>Page name</th>
-                    <th>Description</th>
-                    <th></th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <% @links.each do |link| %>
-                    <tr>
-                      <td>
-                        <%= link.title %>
-                        <%= link_to link.url, target: :_blank, rel: 'noopener' do %>
-                          <span class="nav-link-icon d-md-none d-lg-inline-block">
-                            <%= inline_svg_pack_tag('static/@tabler/icons/icons/link.svg') %>
-                          </span>
-                        <% end %>
-                      </td>
-                      <td>
-                        <%= link.description %>
-                      </td>
-                      <td></td>
-                    </tr>
+<div class="col-12">
```

Comment:
Ezzel a változtatással szétcsúszott a linkeket listázó külön oldal (e.g.: `projects/6/links/` ). Nagyon jó ötlet újra felhasználni egy már megírt kódrészletet, de érdemes odafigyelni, hogy az miért lett már megírva, mire használjuk.

Jelen esetben érdemes az "új" index.html.eb tartalmát kiszervezni egy új partial-be (pl.: links/_table.html.erb` ), amit fel tudsz használni a project oldaláról és a linkek saját listázó oldalán is